Clinical-Genomics/scout
# Truncate protein changes in report

Case report looks ugly for variants which have a very long protein change, see the second pinned variant on this report:
https://scout.scilifelab.se/cust087/F0037995/case_report

This PR truncates the description of protein changes when it exceeds 20 characters. Fix #2308

**How to test**:
1. Locally, modify a variant to include a very long protein change description
1. Pin the variant and see how it looks in master and this branch

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
